rules/windows/registry_event/sysmon_win_reg_telemetry_persistence.yml

```diff
@@ -0,0 +1,29 @@
+title: Registry Persistence Mechanism via Windows Telemetry 
+id: 73a883d0-0348-4be4-a8d8-51031c2564f8
+description: Detects persistence method using windows telemetry 
+status: experimental
+date: 2020/10/16
+references:
+    - https://www.trustedsec.com/blog/abusing-windows-telemetry-for-persistence/
+author: Lednyov Alexey, oscd.community
+tags:
+    - attack.persistence
```

Comment:
Helllo @zipa-original! Thank you for your contribution! Could you please add a technique tag as well?

Reply:
Hello! Unfortunately, I could not find any suitable technique in the ATT&CK matrix. That's why I decided to leave only tactics.

Reply:
I believe that you can use `T1053.005` (Scheduled Task)  as it's gonna be executed by it.

Reply:
I thought about it. But, there is no creation or modification tasks... Do you think it's better to add anyway?

Reply:
"Kinda close" context is better than the absence of context (:

Reply:
You can contribute to the ATT&CK and create the proper technique and then update the rule. What do you think about it?

Reply:
*but for now you can just add `T1053.005` here, @zipa-original

Reply:
Ok